src/main/resources/META-INF/resources/js/filemanager.js

```diff
@@ -129,14 +129,17 @@ function fileManager(_initialData, _$beaconTable, _startURI, _folderPath,
   var expandedFolder;
   var makeEditIcon = function(path, elementAttributes)
   {
-    return "<span class=\"glyphicon glyphicon-pencil\"><a href=\"" +
-    path + "update\" title=\"Edit permissions.\" " +
-    "data-readable=\"" + elementAttributes[6] +
-    "\" data-path=\"" + elementAttributes[9] +
-    "\" data-readgroup=\"" + elementAttributes[5] +
-    "\" data-writegroup=\"" + elementAttributes[4] +
-    "\" data-itemname=\"" + elementAttributes[1] +
-    "\" ></a></span>";
+    return '<span class="glyphicon glyphicon-pencil"><a href="' +
```

Comment:
Make these data-XXX attributes for compliance.  Random attributes are now allowed in HTML.  The jQuery objects can then access them with $("item").data("path").